doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -120,9 +120,27 @@ PSI Variables by Alpha
    energy [H] and correlation correction components [H] for the compound
    method requested through cbs().
 
+.. psivar:: CC ROOT n DIPOLE X
```

Comment:
This all looks good to me. Do you think its important to mention that the ground state info is duplicated? (ie. `CC ROOT 0 DIPOLE X`== `CC DIPOLE X`)